### DISCUSSION TITLE
if support webpack-bundle-analyzer ?

### DISCUSSION THREAD

**Discussion by lemondreamtobe** [parcel-bundler/parcel#4713] [General] [answered]
i want to analyze my bundle, how can i do that?

**Comment by DeMoorJasper** [comment, marked as answer]
This plugin? https://yarnpkg.com/package/parcel-plugin-bundle-visualiser

**Reply by mattrossman** [reply]
@DeMoorJasper It is not compatible with Parcel 2 [(see issue)](https://github.com/gregtillbrook/parcel-plugin-bundle-visualiser/issues/30). They mention [a built-in Parcel 2 bundle analyzer](https://github.com/parcel-bundler/parcel/tree/v2/packages/reporters/bundle-analyzer) that can be enabled by setting the environment variable `PARCEL_BUNDLE_ANALZYZER=1` but it didn't work when I tried it. Any known working Parcel 2 solutions?

**Reply by DeMoorJasper** [reply]
Yes, Parcel 2 has a couple built-in solutions: https://github.com/parcel-bundler/parcel/blob/v2/packages/reporters/bundle-buddy/src/BundleBuddyReporter.js and https://github.com/parcel-bundler/parcel/blob/v2/packages/reporters/bundle-analyzer/src/BundleAnalyzerReporter.js

You can view all the information in their respective PRs: https://github.com/parcel-bundler/parcel/pull/3779 and https://github.com/parcel-bundler/parcel/pull/4648

**Reply by mattrossman** [reply]
Strange, the bundle-analyzer is working for me now when testing on a fresh project after running `export PARCEL_BUNDLE_ANALZYZER=1`. I'm guessing Netlify CLI was clearing out the environment variable on my other project.

**Reply by mattrossman** [reply]
It would be great to have this in the documentation, I'd have no idea about this feature if not for stumbling across the issue / PR.

**Reply by DeMoorJasper** [reply]
@mattrossman definitely, added it to our todo for the Parcel 2 doc-site, thanks!

**Reply by mischnic** [reply]
It's already documented here: https://v2.parceljs.org/features/production/

**Reply by mattrossman** [reply]
Perfect, don't know how I missed that in my search earlier!

**Comment by lemondreamtobe** [comment]
thanks eveyone! nice!
